frontend/e2e/specs/sidePath/unhappy-path.spec.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  importInputWrongProjectFromFile as importUnhappyPathProjectFromFile,
+  importModifiedCorrectConfig as modifiedCorrectProjectFromFile,
+} from '../../fixtures/importFile/unhappy-path-file';
+import { BOARD_METRICS_RESULT, DORA_METRICS_RESULT } from '../../fixtures/createNew/reportResult';
+import { test } from '../../fixtures/testWithExtendFixtures';
+import { clearTempDir } from '../../utils/clearTempDir';
+import { format } from '../../utils/dateTime';
+
+test.beforeAll(async () => {
+  await clearTempDir();
+});
+
+test('Check error UI for pipeline with no org config', async ({ homePage, configStep, metricsStep }) => {
```

Comment:
- please use `should .... when.... given` pattern, like UT
- same as the below test scenior